scripts/util/create-kubernetes-binaries-iso.sh

```diff
@@ -16,9 +16,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
+set -e
```

Comment:
:tr: red flag ;) but fine by me. we allow this script to exit and leave garbage behind. in view of the function of this script fine. (a comment or so would be appropriate, i think)